mixing/mixclient/client.go

```diff
@@ -988,17 +1095,15 @@ func (c *Client) run(ctx context.Context, ps *pairedSessions, madePairing *bool)
 		if sesRun.freshGen {
 			// Generate a new PRNG seed
 			p.prngSeed = new([32]byte)
-			_, err := io.ReadFull(p.rand, p.prngSeed[:])
-			if err != nil {
-				return err
-			}
+			rand.Read(p.prngSeed[:])
```

Comment:
older code may have been checking this for nil before, instead of the `freshGen` field.  I forget and i'm not so curious to dig though tons of earlier revisions to see if that's the case.